ambari-web/latest/src/api/configsApi.ts

```diff
@@ -0,0 +1,270 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { AxiosResponse } from "axios";
+import { ambariApi } from "./config/axiosConfig";
+
+const ConfigsApi = {
+  getServiceConfigurations: async function (
+    stack: string,
+    verison: string,
+    services: string
+  ) {
+    const url = `stacks/${stack}/versions/${verison}/services?StackServices/service_name.in(${services})&fields=configurations/*,configurations/dependencies/*,StackServices/config_types/*`;
+    const response = await ambariApi.request({
+      url: url,
+      method: "GET",
+    });
+    return response.data;
+  },
+  getConfigProperties: async function (
+    stack: string,
+    verison: string,
+    services: string
+  ) {
+    const url = `stacks/${stack}/versions/${verison}/services?StackServices/service_name.in(${services})&fields=configurations/*,configurations/dependencies/*,StackServices/display_name,StackServices/config_types/*&_=1728974996201`;
```

Comment:
what is the use of this static _=1728974996201 parameter

Reply:
@arshadmohammad In the upcoming PRs this will be replaced with the timestamps.

Reply:
ok